### ISSUE TITLE
Webhook support trough the console.

### ISSUE THREAD

**Issue by Doyouwanttomakeacake** [LEGOlord208/DiscordConsole#5]
Currently, the login screen look like this : 
```
DiscordConsole 1.22.4-dev.
Checking for updates...
Latest stable release: 1.22.4.
Download from https://github.com/LEGOlord208/DiscordConsole/releases/tag/1.22.4.
Reading bookmarks...
Please paste your 'token' here, or leave blank for a username/password prompt.
>
```
My suggestion would be to change it to the following : 
```
DiscordConsole 1.22.4-dev.
Checking for updates...
Latest stable release: 1.22.4.
Download from https://github.com/LEGOlord208/DiscordConsole/releases/tag/1.22.4.
Reading bookmarks...
Please paste your 'token' here, or leave blank for a username/password prompt. You can also use a webhook url.
>
```
The idea being you can either input `https://discordapp.com/api/webhooks/301277082145652737/Dw4fJTdKDjEXyJOLlHo7VlzGbsqpz2C5r4gh10MBA8bwmW34h5IYKrzG-tMubfdo9l5P` or just `301277082145652737/Dw4fJTdKDjEXyJOLlHo7VlzGbsqpz2C5r4gh10MBA8bwmW34h5IYKrzG-tMubfdo9l5P`. On a side-note, starting the console with `DiscordConsole --webhook 301277082145652737/Dw4fJTdKDjEXyJOLlHo7VlzGbsqpz2C5r4gh10MBA8bwmW34h5IYKrzG-tMubfdo9l5P` would be working as well. Once logged in with the webhook url, you still have access to a few functions but not all. As an example, features like **big**, **say**, **embed** would work. Also, some features like **name**, **avatar**, .. would be used to change the configuration of the token. All the features that a webhook can't execute won't be displayed in the help; nor they will work.

**Comment by LEGOlord208** [comment]
I think that'd be more suited for a standalone application, but I really like the idea.

Yeah, I'll do it. Why not?

**Comment by LEGOlord208** [comment]
Fixed in 7484f551ffceb9d4c1fe49328e0a3b5ac98d73b5
